doc/decisions/frontend_mocking_tool.md

```diff
@@ -0,0 +1,43 @@
+# Frontend API Mocking Tool
+
+## Problem
+
+We need to select an API mocking tool for our frontend development that allows us to simulate API responses for testing and development purposes.
+
+## Constraints
+
+- The selected tool should seamlessly integrate with our frontend development environment.
+
+## Assumptions
+
+## Considered Alternatives
+
+- **MSW (Mock Service Worker):** [MSW](https://mswjs.io/)
+
+- **Nock:** [Nock](https://github.com/nock/nock)
+
+- **MirageJS:** [MirageJS](https://miragejs.com/)
+
+- **WireMock:** [WireMock](http://wiremock.org/)
```

Comment:
Please add a sentence or two how you know it pros/cons from your own experience (copying info from their homepage is worthless).